yt_dlp/extractor/ichinanalive.py

```diff
@@ -0,0 +1,159 @@
+# coding: utf-8
+from __future__ import unicode_literals
+
+from .common import InfoExtractor
+from ..utils import ExtractorError, str_or_none, traverse_obj, unified_strdate
+from ..compat import compat_str
+
+
+class IchinanaLiveIE(InfoExtractor):
+    IE_NAME = '17live'
+    _VALID_URL = r'https?://(?:www\.)?17\.live/(?:[^/]+/)*(?:live|profile/r)/(?P<id>\d+)'
+    _TESTS = [{
```

Comment:
Add atleast 1 proper test, ie., with all the fields u are returning.

Reply:
This is live stream, and it will be unavailable once it finishes. Do I need it even with that? If so, I'll fix tests.

Reply:
yes

Reply:
As @MinePlayersPE's saying, I'll fill fields with `skip` is set

Reply:
done